system-addon/lib/ActivityStream.jsm

```diff
@@ -261,12 +262,14 @@ this.ActivityStream = class ActivityStream {
     this.store = new Store();
     this.feeds = FEEDS_CONFIG;
     this._defaultPrefs = new DefaultPrefs(PREFS_CONFIG);
+    this._storage = new ActivityStreamStorage(["sectionPrefs", "snippets"]);
   }
 
   init() {
     try {
       this._updateDynamicPrefs();
       this._defaultPrefs.init();
+      this._storage.init();
```

Comment:
what's your reasoning for initializing here?

Reply:
No specific reason just because there's already an `init` function.